pkgs/development/python-modules/jupyter-telemetry/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, python-json-logger
+, jsonschema
+, ruamel_yaml
+, traitlets
+}:
+
+buildPythonPackage rec {
+  pname = "jupyter_telemetry";
+  version = "0.1.0";
```

Comment:
```suggestion
  version = "0.1.0";
  disabled = isPy27;
```

Reply:
I noticed two things: https://github.com/jupyter/telemetry/blob/341f34421d73f11ec5538da8b04a52eb632f7511/setup.py#L33

and 

https://github.com/NixOS/nixpkgs/blob/84cf00f98031e93f389f1eb93c4a7374a33cc0a9/pkgs/development/python-modules/jupyterhub/default.nix#L57

It seems like these is more appropriate?